integration_tests/tests/s3-invoice-end-to-end-tests.ts

```diff
@@ -173,11 +162,18 @@ describe("\n Happy path - Upload valid mock invoice pdf and verify data is seen
 
     // Step 3: Check the view results match the original csv invoice. Hard coded for now based on the csv in the payloads folder.
     const queryString = `SELECT * FROM "btm_billing_curated" where vendor_id = 'vendor_testvendor1' AND year='${"2023"}' AND month='${"03"}' ORDER BY service_name ASC`;
-    const queryId = await startQueryExecutionCommand({
-      databaseName,
-      queryString,
+    const response = await queryAthena(queryString);
+    const queryObjects: BillingCurated[] = response.map((item) => {
+      return {
+        vendor_id: item.vendor_id,
+        vendor_name: item.vendor_name,
+        service_name: item.service_name,
+        quantity: item.quantity,
+        price: item.price,
+        year: item.year,
+        month: item.month,
+      };
     });
```

Comment:
Why do this map, are there keys on the `item`s that you don't want?

Reply:
types were different so used map, but now changed to generics type as per suggestion